plugins/GSdx/GSOsdManager.cpp

```diff
@@ -149,13 +172,13 @@ void GSOsdManager::RenderGlyph(GSVertexPT1* dst, glyph_info g, float x, float y,
 	++dst;
 }
 
-void GSOsdManager::RenderString(GSVertexPT1* dst, std::string msg, float x, float y, float m_sx, float m_sy, uint32 color) {
-	for(const unsigned char* p = (unsigned char*)msg.c_str(); *p; p++) {
-		RenderGlyph(dst, c_info[*p], x, y, m_sx, m_sy, color);
+void GSOsdManager::RenderString(GSVertexPT1* dst, std::u32string msg, float x, float y, float m_sx, float m_sy, uint32 color) {
+	for(auto c : msg) {
```

Comment:
why not `const auto & c` ? Maybe useless because basic type but it feels more robust.